test/debugger/test-debugger-stop.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+var common = require('../common');
+var debug = require('_debugger');
+
+var fork = require('child_process').fork;
+
+var scriptToDebug = common.fixturesDir + '/debug-connect.js';
+
+var nodeProcess;
+
+var failed = true;
+try {
+  process._debugProcess(process.pid);
+  failed = false;
+} finally {
+  // At least TRY not to leave zombie procs if this fails.
+  if (failed)
+    process.kill();
```

Comment:
Instead of this, please do `assert.fail`. In face, I would recommend doing

```js
try {
  process._debugProcess(process.pid);
} catch(ex) {
  assert.fail('Debugging should not fail');
}
```